guides/common/modules/proc_synchronizing-repositories-by-using-web-ui.adoc

```diff
@@ -0,0 +1,12 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="synchronizing-repositories-by-using-web-ui"]
+= Synchronizing repositories by using {ProjectWebUI}
+
+[role="_abstract"]
+You can synchronize repositories one-time by using {ProjectWebUI}.
```

Comment:
I am a bit unsure about the wording of "one-time". (non-blocking) same in CLI procedure.

Reply:
Either you suggest an alternative or I'm keeping it :)

Reply:
I have nothing!